src/components/CheckoutSummary/CheckoutSummary.test.js

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import renderer from "react-test-renderer";
+import { ComponentsProvider } from "@reactioncommerce/components-context";
+import componentsContext from "componentsContext";
+import CheckoutSummary from "./CheckoutSummary";
+
+const testCart = {
+  checkout: {
+    summary: {
+      itemTotal: {
+        displayAmount: "$118"
+      },
+      total: {
+        displayAmount: "$118"
+      }
+    }
+  },
+  totalItemQuantity: 3,
+  items: [
+    {
+      _id: "123",
+      attributes: [
+        { label: "Color", value: "Red" },
+        { label: "Size", value: "Medium" }
+      ],
+      compareAtPrice: {
+        displayAmount: "$45.00"
+      },
+      currentQuantity: 3,
+      imageURLs: {
+        small: "//placehold.it/150",
+        thumbnail: "//placehold.it/100"
+      },
+      isLowQuantity: true,
+      price: {
+        displayAmount: "$20.00"
+      },
+      productSlug: "/product-slug",
+      productVendor: "Patagonia",
+      title: "A Great Product",
+      quantity: 2
+    },
+    {
+      _id: "456",
+      attributes: [
+        { label: "Color", value: "Black" },
+        { label: "Size", value: "10" }
+      ],
+      currentQuantity: 500,
+      imageURLs: {
+        small: "//placehold.it/150",
+        thumbnail: "//placehold.it/100"
+      },
+      isLowQuantity: false,
+      price: {
+        displayAmount: "$78.00"
+      },
+      productVendor: "Nike",
+      productSlug: "/product-slug",
+      title: "Another Great Product",
+      quantity: 1
+    }
+  ]
+};
+
+test("basic snapshot", () => {
+  const component = renderer.create((
+    <ComponentsProvider value={componentsContext}>
+      <CheckoutSummary
```

Comment:
lets add stubs for these props: 
```
hasMoreCartItems: PropTypes.bool,
    loadMoreCartItems: PropTypes.func,
    onChangeCartItemsQuantity: PropTypes.func,
    onRemoveCartItems: PropTypes.func,
```
Getting an error when I click the remove or increment quantities.

Reply:
How are you testing the click in a snapshot test?

Reply:
![cart_checkout_error](https://user-images.githubusercontent.com/486340/44229094-dac89e00-a14b-11e8-9ba5-ffe9a38a3222.png)

Just clicking on the elements of the summary

Reply:
I see, so it's not the test, its the actual component in the page.

Reply:
Yes, in the actual component in the checkout page component.